README.md

```diff
@@ -1098,6 +1102,66 @@ const options: TelescopeOptions = {
 };
 ```
 
+## Name Mappers Configuration
```

Comment:
I think the top level of this should be sth like:  Helper Functions. And include this part inside.

And under the Options of Helper Functions, We can have a reference like others did:

See [Helper Functions](xxx) for more info.

Reply:
Done, can you please take another look, thank you.